### ISSUE TITLE
Will TuiGAN still working well on image of high resolution

### ISSUE THREAD

**Issue by Nick-Sun1018** [linjx-ustc1106/TuiGAN-PyTorch#10]
If I want to implement TuiGAN on task like Dehazing or low-light enhancement, will TuiGAN generate a high resolution image?

**Comment by linjx-ustc1106** [comment]
Yes, we have conducted several experiments on high-resolution images, such as photorealistic style transfer. Also, in the second row of figure 5 in our paper, the result shows that our model have certain ability for low-light enhancement.
